modules/dop/providers/issue/sync/service.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sync
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/erda-project/erda-infra/base/logs"
+	"github.com/erda-project/erda-proto-go/dop/issue/sync/pb"
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/modules/dop/dao"
+	issueSvc "github.com/erda-project/erda/modules/dop/services/issue"
+	"github.com/erda-project/erda/pkg/common/apis"
+)
+
+type IssueSyncService struct {
```

Comment:
I think patch update is part of issue's CRUD, is there a need to define a separate provider? what do you think? @sfwn

Reply:
we update labels and iteration in this api